persistence/src/main/java/org/cbioportal/genome_nexus/persistence/VariantAnnotationRepository.java

```diff
@@ -32,12 +32,11 @@
 
 package org.cbioportal.genome_nexus.persistence;
 
-import org.cbioportal.genome_nexus.persistence.internal.VariantAnnotationRepositoryCustom;
 import org.cbioportal.genome_nexus.model.VariantAnnotation;
 import org.springframework.data.mongodb.repository.MongoRepository;
 
 /**
  * @author Benjamin Gross
  */
 public interface VariantAnnotationRepository
-    extends MongoRepository<VariantAnnotation, String>, VariantAnnotationRepositoryCustom {}
+    extends MongoRepository<VariantAnnotation, String>, GenericMongoRepository {}
```

Comment:
now `VariantAnnotationRepository` implements `GenericMongoRepository`

Reply:
Doesn't this save the object as a String, instead of before as a JSON object?